### ISSUE TITLE
RCTView generated view config for validAttributes does not match native, missing: accessibilityValue

### ISSUE THREAD

**Issue by Ivanov89** [expo/expo#9540]
## 🐛 Bug Report

Environment: 

` Expo CLI 3.21.5 environment info:
    System:
      OS: macOS 10.15.6
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 12.12.0 - /usr/local/bin/node
      Yarn: 1.19.1 - /usr/local/bin/yarn
      npm: 6.11.3 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 4.0 AI-193.6911.18.40.6514223
      Xcode: 11.6/11E708 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^38.0.9 => 38.0.9 
      react: 16.9.0 => 16.9.0 
      react-dom: 16.9.0 => 16.9.0 
      react-native: https://github.com/expo/react-native/archive/sdk-37.0.1.tar.gz => 0.61.4 
      react-navigation: ^3.11.1 => 3.13.0 
    npmGlobalPackages:
      expo-cli: 3.21.5
`

After `expo start` the following error occurs:

<img width="312" alt="Screenshot 2020-08-04 at 12 10 55" src="https://user-images.githubusercontent.com/4004486/89275986-b6716f00-d64b-11ea-8a11-9e474fbef248.png">

Tried to reset cache on expo, yarn and npm but doesn't work.

Your feedback will be appreciated!
Thanks!

**Comment by Ivanov89** [comment]
It was because I didn't quit the iOS Simulator ! :)
You can close the issue if you want !

**Comment by ldub** [comment]
I am having this issue on both android and ios physical devices, no simulator running. I run `expo start` on my app and as soon as its loaded I get this exact same error.

```
❯ expo diagnostics

  Expo CLI 3.27.13 environment info:
    System:
      OS: macOS 10.15.5
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 12.16.3 - ~/.nvm/versions/node/v12.16.3/bin/node
      Yarn: 1.22.4 - ~/.nvm/versions/node/v12.16.3/bin/yarn
      npm: 6.14.4 - ~/.nvm/versions/node/v12.16.3/bin/npm
    SDKs:
      iOS SDK:
        Platforms: iOS 14.0, DriverKit 19.0, macOS 10.15, tvOS 14.0, watchOS 7.0
    IDEs:
      Xcode: 12.0/12A7209 - /usr/bin/xcodebuild
    npmPackages:
      expo: ~38.0.10 => 38.0.10
      react: ~16.13.1 => 16.13.1
      react-dom: ~16.13.1 => 16.13.1
      react-native: https://github.com/expo/react-native/archive/sdk-37.0.1.tar.gz => 0.61.4
    npmGlobalPackages:
      expo-cli: 3.27.13
    Expo Workflow: managed
```

@byCedric could you suggest any steps for diagnosing this error?

**Comment by listiani13** [comment]
Is there any chance you're having `expo-linking` in your dependencies? My friend ran into this problem after doing `expo install expo-linking` @ldub 
cc @liukinsay97

**Comment by stevemer** [comment]
Ran into the same issue:

steves-mbp:anvil-mobile mala$ expo diagnostics

  Expo CLI 3.28.5 environment info:
    System:
      OS: macOS 10.15.7
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 14.4.0 - /usr/local/bin/node
      Yarn: 1.22.4 - /usr/local/bin/yarn
      npm: 6.14.4 - /usr/local/bin/npm
    SDKs:
      iOS SDK:
        Platforms: iOS 13.7, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    IDEs:
      Android Studio: 4.0 AI-193.6911.18.40.6514223
      Xcode: 11.7/11E801a - /usr/bin/xcodebuild
    npmPackages:
      expo: ^39.0.4 => 39.0.4 
      react: ~16.9.0 => 16.9.0 
      react-dom: ~16.9.0 => 16.9.0 
      react-native: https://github.com/expo/react-native/archive/sdk-37.0.1.tar.gz => 0.61.4 
      react-native-web: ~0.11.7 => 0.11.7 
    npmGlobalPackages:
      expo-cli: 3.28.5
    Expo Workflow: managed

**Comment by stevemer** [comment]
RCTView generated view config for validAttributes does not match native, missing: accessibilityValue
* http://192.168.86.20:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false&minify=false:68297:29 in <unknown>
* http://192.168.86.20:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false&minify=false:104924:24 in error
* http://192.168.86.20:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false&minify=false:48969:24 in <unknown>
* [native code]:null in forEach
* http://192.168.86.20:19001/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false&minify=false:48965:76 in verifyComponentAttributeEquivalence

**Comment by brentvatne** [comment]
@stevemer - you're using react-native sdk-37.0.0 / 0.61.4 with expo sdk 39, look at the output of your diagnostics command.

**Comment by stevemer** [comment]
@brentvatne Sorry to be an idiot, but I don't understand why that's a problem or how to fix it.

**Comment by liukinsay97** [comment]
> @stevemer - you're using react-native sdk-37.0.0 / 0.61.4 with expo sdk 39, look at the output of your diagnostics command.

Yeah I just realized this too...
this can be fixed by updating the expo react native to version 39.0.3
simply do `expo install react-native` or just change the react native to [version 39.0.0](https://expo.io/--/api/v2/versions)
`"react-native": "https://github.com/expo/react-native/archive/sdk-39.0.3.tar.gz"` and install it

**Comment by ldub** [comment]
My issue was also related resolved by updating to sdk39. It was also related to the ios14 launch.

**Comment by skavor** [comment]
> It was because I didn't quit the iOS Simulator ! :) You can close the issue if you want !

How ???

**Comment by rahul8x** [comment]
i still show same error 


buildscript {
    ext {
        buildToolsVersion = "31.0.0"
        minSdkVersion = 21
        compileSdkVersion = 31
        targetSdkVersion = 31
        kotlinVersion = "1.6.21"
    }
    repositories {
        google()
        jcenter()
    }
    dependencies {
        classpath("com.android.tools.build:gradle:3.6.3")
        classpath 'com.google.gms:google-services:4.3.5'
        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
}

**Comment by ArdineJ** [comment]
same here
